.github/workflows/continuous-integration.yml

```diff
@@ -10,63 +10,69 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
+
     steps:
       - name: "Checkout"
         uses: actions/checkout@master
 
-      - name: "Install PHP 7.4"
-        run: |
-          sudo add-apt-repository ppa:ondrej/php
-          sudo apt-get update
-          sudo apt-get install php7.4 php7.4-mbstring php7.4-xml
+      - name: "Set default PHP version"
+        run: sudo update-alternatives --set php /usr/bin/php${{ matrix.php-version }}
```

Comment:
Use built-in PHP 7.4

Reply:
https://github.com/woohoolabs/zen/blob/3.0/.github/workflows/continuous-integration.yml uses shivammathur/setup-php for the similar purpose, and I'd prefer using the same approach in master as well.

Reply:
`ubuntu-latest` image already contain zillions of software including PHP 7.* and 8.0
Using a separate action is complicated, redundant and error prone.

Reply:
Yes, but for example it doesn't contain latest dev version of PHP (8.1), while this action already supports it. Also, it makes it very easy to enable/disable extensions (like xdebug, see example in the v3 branch)

Reply:
GitHub Actions being community driven is very bad. Errors, bugs are coming from multiple authors with different mindsets.
I am Travis CI guy :)

Reply:
I do use zen, nor I am a developer. So I leave it up to you.

Reply:
I research things! e.g. DI containers https://github.com/szepeviktor/simple-di-container/issues